ctt/process_dependencies.py

```diff
@@ -465,6 +465,21 @@ def process_job(processing_job: processing_model.ProcessingJob):
             oci_client=oci_client,
         )
 
+        if (extra_tags := processing_job.extra_tags):
```

Comment:
nit: I guess this is somewhat personal preference but these parentheses are not required

Reply:
removed